invenio/ext/template/__init__.py

```diff
@@ -170,6 +170,12 @@ def _sentences(value, limit, separator='. '):
         """Return first `limit` number of sentences ending by `separator`."""
         return separator.join(value.split(separator)[:limit])
 
+    @app.template_filter('characters')
+    def _characters(value, limit, extension=' [...]'):
```

Comment:
IMHO `truncate` http://jinja.pocoo.org/docs/dev/templates/#truncate should do the job